aiosmtpd/smtp.py

```diff
@@ -317,21 +321,27 @@ def _set_rset_state(self):
                 # The connection got reset during the DATA command.
                 # XXX If handler method raises ConnectionResetError, we should
                 # verify that it was actually self._reader that was reset.
-                log.info('Connection lost during _handle_client()')
+                log.info('%r Connection lost during _handle_client()',
+                         self.session.peer)
                 self._writer.close()
                 raise
             except Exception as error:
                 try:
                     status = await self.handle_exception(error)
-                    await self.push(status)
-                except Exception as error:
+                except Exception as error2:
```

Comment:
Would this be better as `inner_error`? `error2` just seems a bit ambiguous to me :thinking:

Reply:
Of course no problem (I'm just used to call them ex, ex2 for my personal codes so are used to that).